sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_event_v1/init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS
+  clients_daily_event_v1
```

Comment:
Not sure if we want to maintain a init.sql file for this query? I'm guessing it will change as new events get added.

Currently, the init.sql is missing `n_created_pictureinpicture` and `n_viewed_protection_report`

Reply:
yeah, I don't think this is the right place for an `init.sql`. The table doesn't self-join, so `query.sql` should be sufficient on its own.